### ISSUE TITLE
Use Config_UnnumberedElements

### ISSUE THREAD

**Issue by tomjw64** [openstax/cnx-recipes#1060]
Convert the library to use `$Config_UnnumberedElements` rather than `$Config_UnnumberedExercises` and `$Config_UnnumberedEquations`

**Comment by openstaxalina** [comment]
Discussed with @Adrianltd and @tomjw64 -- the CSS output should not change.  Therefore, there should be no need for regression testing.

**Comment by Adrianltd** [comment]
We previously had `unnumbered equations` being called in the generator at pass 1 and `unnumbered exercises` being called in the generator at pass 3.
During the `precalculus recipe` creation I needed to add `unnumbered figures`, so instead - as we have discussed in https://github.com/openstax/cnx-recipes/pull/836 - I've unified all that 3 options to be `unnumbered elements`.
@tomjw64 we still might need to do some regression testing. Because there's less uses of unnumbered equations than unnumbered exercises, I think it's safer to put that `$Config_UnnumberedElements` to pass 3. Then there's only `physics` and `ap-physics` to be tested. Although I don't see any possible issues caused by that, there's nothing happening with equations between pass 1 and 3.
